pkg/flowaggregator/exporter/clickhouse_test.go

```diff
@@ -55,13 +55,14 @@ func TestClickHouse_UpdateOptions(t *testing.T) {
 		},
 		ClickHouseCommitInterval: 8 * time.Second,
 	}
-	chInput := buildClickHouseInput(opt)
-	chExportProcess, err := clickhouseclient.NewClickHouseClient(chInput, uuid.New().String())
+	chConfig := buildClickHouseConfig(opt)
+	chExportProcess, err := clickhouseclient.NewClickHouseClient(chConfig, uuid.New().String())
 	require.NoError(t, err)
-	clickHouseExporter := ClickHouseExporter{chInput: &chInput, chExportProcess: chExportProcess}
+	clickHouseExporter := ClickHouseExporter{chConfig: &chConfig, chExportProcess: chExportProcess}
 	clickHouseExporter.Start()
-	assert.Equal(t, clickHouseExporter.chExportProcess.GetDsn(), "tcp://clickhouse-clickhouse.flow-visibility.svc:9000?username=default&password=default&database=default&debug=true&compress=false")
+	assert.True(t, reflect.DeepEqual(clickHouseExporter.chExportProcess.GetClickHouseConfig(), chConfig))
```

Comment:
`assert.Equal` will actually call `reflect.DeepEqual` if you want to keep using that